website/data/version.js

```diff
@@ -1 +1 @@
-export default '0.2.0'
+export default { boundary: '0.2.0', 'boundary-desktop': '1.0.0' }
```

Comment:
Calling out changes to the version file.

Reply:
I might export these are separate variables - this is what is done on other docs sites. like:

```js
export const VERSION = '0.2.0'
export const DESKTOP_VERSION = '1.0.0'
```

Reply:
Done, thank you!